src/libs/actions/Policy/execPolicyWriteCommand.ts

```diff
@@ -0,0 +1,30 @@
+import type {OnyxCollection} from 'react-native-onyx';
+import Onyx from 'react-native-onyx';
+import * as API from '@libs/API';
+import type {ApiRequestCommandParameters, WriteCommand} from '@libs/API/types';
+import * as NetworkStore from '@libs/Network/NetworkStore';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Policy} from '@src/types/onyx';
+import type {OnyxData} from '@src/types/onyx/Request';
+
+let allPolicies: OnyxCollection<Policy>;
+
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY,
+    waitForCollectionCallback: true,
+    callback: (value) => (allPolicies = value),
+});
+
+function execPolicyWriteCommand<TCommand extends WriteCommand>(commandName: TCommand, params: ApiRequestCommandParameters[TCommand], onyxData: OnyxData = {}, policyID?: string) {
+    const extendedParams: (typeof params & {lastModified?: string | number}) | null = params && {...params};
+    if (NetworkStore.isOffline() && extendedParams) {
```

Comment:
As we save requests to queue, so it's possible that requests are pending in the queue and network is changing to offline at the same time. Is it better to always pass `lastModified` and just let the backend to handle the order of write conflicts?

https://github.com/Expensify/App/blob/6fe216e2a46f2fbb8a33808d354f6696c8cbdab2/src/libs/API/index.ts#L88-L93

Reply:
@roryabraham Can you share how `lastModified` works in BE?

Reply:
We only allow stale policy updates coming from the same client. This mainly supports offline policy updates in newDot, where when multiple offline requests are made, they can be successful

Does that answer your question? I think any time we're making a policy write command it would be safe to pass `lastModified`. We don't need to do it only when offline, as is implemented here